apps/aecore/include/blocks.hrl

```diff
@@ -1,6 +1,6 @@
 -include("pow.hrl").
 
--define(PROTOCOL_VERSION, 19).
+-define(PROTOCOL_VERSION, 20).
```

Comment:
Already bumped since 0.18.0 so unneeded. Though does not harm.

Also this PR does not impact consensus, right? So only DB_VERSION bump needed. Though does not harm.

Reply:
Ok, I was not sure what is the relation between protocol version and db bump, so did that just in case.